apps/src/applab/dropletConfig.js

```diff
@@ -100,9 +100,9 @@ export var blocks = [
 
   {func: 'createCanvas', parent: api, category: 'Canvas', paramButtons: { minArgs: 1, maxArgs: 3 }, paletteParams: ['id','width','height'], params: ['"id"', DEFAULT_WIDTH, DEFAULT_HEIGHT] },
   {func: 'setActiveCanvas', parent: api, category: 'Canvas', paletteParams: ['id'], params: ['"id"'], dropdown: { 0: idDropdownWithSelector("canvas") } },
-  {func: 'line', parent: api, category: 'Canvas', paletteParams: ['x1','y1','x2','y2'], params: ["0", "0", "160", "240"] },
-  {func: 'circle', parent: api, category: 'Canvas', paletteParams: ['x','y','radius'], params: ["160", "240", "100"] },
-  {func: 'rect', parent: api, category: 'Canvas', paletteParams: ['x','y','width','height'], params: ["80", "120", "160", "240"] },
+  {func: 'line', parent: api, category: 'Canvas', paletteParams: ['x1','y1','x2','y2'], params: ["0", "0", "160", DEFAULT_HEIGHT/2] },
```

Comment:
The `DEFAULT_HEIGHT` constant in this file is sort of unfortunate - it's got the 320 and 480 hard-coded, when they also exist in [applab/constants.js](https://github.com/code-dot-org/code-dot-org/blob/23b5c6f3de57fe106a63e441ec7760f6cbfb2c35/apps/src/applab/constants.js#L8-L8), which this file _already_ imports.  How about we do a little refactoring and deduplicate those magic numbers?